File: src/test/java/com/robohorse/gpversionchecker/utils/AssetsReader.java
package com.robohorse.gpversionchecker.utils;

import junit.framework.Assert;

import org.apache.commons.io.IOUtils;

import java.io.IOException;
import java.io.InputStream;
import java.io.StringWriter;

/**
 * Created by vadim on 17.07.16.
 */
public class AssetsReader {

    public String read(String name) throws IOException, NullPointerException {
        String content = null;
        try {
            content = readFile(name);
        } catch (IOException | NullPointerException e) {
            Assert.fail(name + " not found");
        }

        Assert.assertNotNull("Content is empty", content);
        return content;
    }

    private String readFile(String name) throws IOException, NullPointerException {
        InputStream inputStream = this.getClass().getClassLoader().getResourceAsStream(name);
        StringWriter writer = new StringWriter();
        IOUtils.copy(inputStream, writer, "UTF-8");
        IOUtils.closeQuietly(inputStream);
        return writer.toString();
    }
}

File: gpversionchecker/src/test/kotlin/com/robohorse/gpversionchecker/utils/AssetsReader.kt
package com.robohorse.gpversionchecker.utils

import org.apache.commons.io.IOUtils
import java.io.IOException
import java.io.StringWriter

class AssetsReader {

    fun read(name: String) = try {
        readFile(name)
    } catch (e: IOException) {
        e.printStackTrace()
        null
    }

    private fun readFile(name: String): String {
        val inputStream = this.javaClass.classLoader!!.getResourceAsStream(name)
        val writer = StringWriter()
        IOUtils.copy(inputStream, writer, "UTF-8")
        IOUtils.closeQuietly(inputStream)
        return writer.toString()
    }
}
